src/ducks/mobile/utils.js

```diff
@@ -41,6 +42,7 @@ export const setBarTheme = theme => {
   }
 
   if (cozy.bar && cozy.bar.setTheme) {
-    cozy.bar.setTheme(theme)
+    const overrides = barOverrides[theme]
+    cozy.bar.setTheme(theme, overrides)
```

Comment:
If the theme does not exist, it should throw ? What happens ?

Reply:
IMHO it should set the default theme but not fail. We can add a warning in the console.

Reply:
OK for me.

Reply:
But it should be implemented in the cozy-bar since it knows its available themes.